lib/src/backend.rs

```diff
@@ -156,18 +156,44 @@ content_hash! {
         // TODO: Store e.g. CommitId here too? Labels (theirs/ours/base)? Would those still be
         //       useful e.g. after rebasing this conflict?
         pub value: TreeValue,
+        pub negative: bool,
```

Comment:
Whenever I see `struct S { data: T, some_tag: bool }` I'm like "ohh this should be a enum":
```rust
pub enum Conflict {
  Positive(TreeValue),
  Negative(TreeValue),
}
```
I assume that this would probably require more refactors, but this is way cleaner in the end 🤷